addons/rook/template/base/install.sh

```diff
@@ -99,14 +99,10 @@ function rook() {
     fi
 
     if ! kubectl -n rook-ceph get pod -l app=rook-ceph-rgw -o jsonpath='{.items[0].status.phase}' 2>/dev/null | grep -q Running ; then
-        semverParse "$ROOK_VERSION"
-        # shellcheck disable=SC2154
-        local rook_major_minor_version="${major}.${minor}"
-
         printf "\n\n%bRook Ceph 1.4+ requires a secondary, unformatted block device attached to the host.%b\n" "$GREEN" "$NC"
         printf "%bIf you are stuck waiting at this step for more than two minutes, you are either missing the device or it is already formatted.%b\n" "$GREEN" "$NC"
         printf "\t%b * If it is missing, attach it now and it will be picked up; or CTRL+C, attach, and re-start the installer%b\n" "$GREEN" "$NC"
-        printf "\t%b * If the disk is attached, try wiping it using the recommended zap procedure: https://rook.io/docs/rook/v%s/ceph-teardown.html#zapping-devices%b\n\n" "$GREEN" "$rook_major_minor_version" "$NC"
+        printf "\t%b * If the disk is attached, try wiping it using the recommended zap procedure: https://rook.io/docs/rook/v1.10/Storage-Configuration/ceph-teardown/?h=zap#zapping-devices%b\n\n" "$GREEN" "$NC"
```

Comment:
the problem here is that we will need to manually update this one for future releases > 1.10 (i.e. 1.11). However, I am fine with that.